R/type-date-time.R

```diff
@@ -209,6 +209,11 @@ vec_cast.POSIXt.Date <- function(x, to) {
   as.POSIXct(as.character(x), tz = tzone(to))
 }
 #' @export
+#' @method vec_cast.POSIXt POSIXlt
+vec_cast.POSIXt.POSIXlt <- function(x, to) {
```

Comment:
I think this (and the method below), should be `vec_cast.POSIXct`. Would you mind fixing?